docs/hugo/content/introduction/metrics.md

```diff
@@ -0,0 +1,45 @@
+# ASOv2 Metrics
+
+Prometheus metrics are exposed for ASOv2 which can be helpful in diagnosability and tracing.
+The metrics exposed fall into two groups: Azure based metrics and reconciler metrics (from `controller-runtime`) 
+combined.
+
+## Enabling the metrics
+
+By default, metrics scraping is turned on for ASOv2. It can be turned off using the Helm chart `--set metrics.enable` or by omitting `--metrics-addr` flag from the Deployment yaml.
+
+### ASOv2 Helm Chart
+
+While installing the Helm chart, we can turn the metrics ON and OFF and set the metrics expose address using the 
+below settings. Also, we can change the settings inside `values.yaml` file for ASOv2 Helm chart.
+
+   ```
+   --set metrics.enable=true/false (default: true)
+   --set metrics.address=127.0.0.1:8080 (default)
+   ```
+
+### Deployment Yaml
+
+In the deployment yaml, we can turn OFF the metrics by omitting the `metrics-addr` flag below and also set the metrics 
```

Comment:
minor: fix this sentence. You sorta same the same thing twice. I think you can just remove the "and also ..." part.